Makefile

```diff
@@ -17,9 +17,17 @@ ifeq (, $(shell which ginkgo))
 	go get github.com/onsi/ginkgo/ginkgo
 endif
 
-
 test: test-tools
-	ginkgo -r ./internal/* ./cmd/*
+	ginkgo -r $(GINKGO_OPTIONS) ./internal/* ./cmd/*
+
+test-coverage:
+test-coverage: GINKGO_OPTIONS ?= --cover
+test-coverage: test
+	gover
+	go tool cover -html gover.coverprofile
+
+test-coverage-clean:
+	git clean -fx
```

Comment:
This command is very unsafe - it removes git-ignored IDE project files as well. Please find other way to clean-up coverage artifacts.